repo-scripts/api-documenter/README.md

```diff
@@ -3,3 +3,14 @@
 It is a fork of [API Documenter](https://github.com/microsoft/rushstack/tree/master/apps/api-documenter)
 It reads the *.api.json data files produced by [API Extractor](https://api-extractor.com/),
 and then generates files in [Markdown](https://en.wikipedia.org/wiki/Markdown) format suitable for displaying in Firebase Devsite.
+
+## Generate toc for Firebase devsite
```

Comment:
You might want to document  the `-g3-path` flag here as well.

Reply:
-p is the short flag for --g3-path, but good point I will list both of them